frontend/device/android/device.lua

```diff
@@ -91,7 +92,12 @@ function Device:initNetworkManager(NetworkMgr)
 end
 
 function Device:retrieveNetworkInfo()
-    return android.getNetworkInfo()
+    local ssid, ip, gw = string.match(android.getNetworkInfo(), "(.*);(.*);(.*)")
+    if ip == 0 or gw == 0 then
+        return _("Not connected")
+    else
+        return _(string.format("Conected to %s\n ip address: %s\n gateway: %s", ssid, ip, gw))
```

Comment:
Please use util.format instead.

PS Connected & IP.

Reply:
I can't find util.format.

Reply:
Sorry, I messed up in between don't use string.format and please use util.template. :blush: 

```lua
local T = require("ffi/util").template
```

```lua
        return T(_("Connected to %1\n IP address: %2\n gateway: %3"), ssid, ip, gw)
```

string.format doesn't allow for free positioning of the provided strings/numbers, which is generally okay between Indo-European languages but otherwise bad for localization.